src/Plugin/Requirement/Requirement/ApigeeEdgeDeveloperSync.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\apigee_m10n\Plugin\Requirement\Requirement;
+
+use Drupal\apigee_edge\Job\Job;
+use Drupal\requirement\Plugin\RequirementBase;
+
+/**
+ * Check if the developer sync job has been run.
+ *
+ * @Requirement(
+ *   id="apigee_edge_developer_sync",
+ *   group="apigee_edge",
+ *   label="Developer Sync",
+ *   description="Sync Drupal users with Apigee Edge developers.",
+ *   severity="recommendation",
+ *   form="\Drupal\apigee_edge\Form\DeveloperSyncForm",
+ *   action_button_label="Sync developers",
+ *   dependencies={
+ *      "apigee_edge_connection",
+ *   }
+ * )
+ */
+class ApigeeEdgeDeveloperSync extends RequirementBase {
```

Comment:
Same with this class, this seems to be something that should be part of Apigee Edge module.

Reply:
See https://github.com/apigee/apigee-m10n-drupal/pull/191#discussion_r371066752